packages/components/tests/dummy/app/styles/components/dummy-component-props.scss

```diff
@@ -49,15 +49,28 @@
         background-color: #e4e4e4;
       }
     }
+  }
 
-    .default {
-      display: inline-block;
-      padding: 0.125rem 0.25rem;
-      border-radius: 4px;
-      font-weight: 700;
-      color: #0c56e9;
-      background-color: #cce3fe;
-    }
+  // Indicators / Tags
+  .default,
+  .required {
+    border-radius: 4px;
+    font-weight: 700;
+    display: inline-block;
+    padding: 0.125rem 0.25rem;
+  }
```

Comment:
Moved default styles here so they could be grouped with the similar styles for "required" which I added. This changes the element the styles are scoped to which didn't seem to me that it would be a problem but let me know if I'm wrong.

Reply:
Thanks, I'll update it to repeat the shared styles.

Commentary: In general, I usually try to standardize visual styling of similar elements where it makes sense and try to avoid duplicating code. Typically, I would either separate out the common styles so that they could be referenced by both elements or keep what seem to be similar items like this grouped. Then if a need arose in the future to visually differentiate them I would refactor at that time.